std/container/rbtree.d

```diff
@@ -20,6 +20,7 @@ Authors: Steven Schveighoffer, $(WEB erdani.com, Andrei Alexandrescu)
 module std.container.rbtree;
 
 import std.functional : binaryFun;
+import std.format;
```

Comment:
Hm... Move that to toString method itself? Since RB-Tree is a template that should cut down on number of imports if RB-Tree is not instantiated but  just imported.